README.md

```diff
@@ -1,8 +1,30 @@
 Embox [![Build Status](https://travis-ci.org/embox/embox.svg?branch=master)](https://travis-ci.org/embox/embox) [![Coverity Scan Build Status](https://scan.coverity.com/projects/700/badge.svg)](https://scan.coverity.com/projects/700)
 =====
 
-Embox is a configurable operating system kernel designed for resource
-constrained and embedded systems.
+Embox is a configurable real-time operating system designed for resource constrained and embedded systems.
+
+Key features
+---------------
+* Support POSIX
+* Support C++
+* Support file system
```

Comment:
List of supported file systems would look very natural here

Reply:
Done